Makefile

```
@@ -116,11 +116,15 @@ help:  ## Display this help
 
 .PHONY: test
 test: ## Run tests.
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test ./... $(TEST_ARGS)
+
+.PHONY: test-verbose
+test-verbose: ## Run tests with verbose settings.
 	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./... $(TEST_ARGS)
 
 .PHONY: test-cover
-test-cover: ## Run tests with code coverage and code generate  reports
-	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v -coverprofile=out/coverage.out ./... $(TEST_ARGS)
+test-cover: ## Run tests with code coverage and code generate reports.
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -coverprofile=out/coverage.out ./... $(TEST_ARGS)
```

```suggestion
	TEST_ARGS="$(TEST_ARGS) -coverprofile=out/coverage.out" $(MAKE) test
```

Something like that?

---

Tested locally and works :+1: